pkg/lib/cockpit.d.ts

```diff
@@ -118,7 +118,7 @@ declare module 'cockpit' {
     /* === cockpit.spawn ============================= */
 
     interface Spawn<T> extends DeferredPromise<T> {
-        input(message: T, stream?: boolean): DeferredPromise<T>;
+        input(message?: T | null, stream?: boolean): DeferredPromise<T>;
```

Comment:
Ah, this is for the case when `stream` is false, i.e. this means "close stdin"?

Reply:
It could theoretically also be used with `stream=true` but ya, this basically lets you call `input()` to mean `EOF`.  I tested that today, and it is indeed what happens.